servicemigration/context.py

```diff
@@ -105,7 +105,9 @@ def commit(self, filename=None):
         unmodifiedServices = []
         for svc in self.services:
             serial = service.serialize(svc)
-            serial["Version"] = self.version
+            # Only update version if it is non-empty
```

Comment:
Please remove lines 108-110. I spoke to Ian and we can't think of a use-case where it makes sense for the migration of single service to modify the version info.  In all cases where we migrate Core, RM and USCPM service definitions, the versioning changes occur outside of the service migration covered by this SDK.

Also, please run `make test` to verify that the unit-tests pass (otherwise, we can't merge the PR).  I think there's one or two tests that deal with setting the Version property.